generate.js

```diff
@@ -59,6 +59,7 @@ const typescriptTemplate = {
     '@types/node': cliPkg.devDependencies['@types/node'],
     '@types/tap': cliPkg.devDependencies['@types/tap'],
     concurrently: cliPkg.devDependencies.concurrently,
+    'fastify-tsconfig': cliPkg.devDependencies['fastify-tsconfig'],
```

Comment:
I think we are missing this in package.json.

Reply:
We added it when we added the `generate-plugin` command.